opennms-dao/pom.xml

```diff
@@ -114,7 +114,6 @@
     <dependency>
       <groupId>org.opennms.features.collection</groupId>
       <artifactId>org.opennms.features.collection.persistence.rrd</artifactId>
-      <scope>test</scope>
```

Comment:
Is this necessary? Having openms-dao depend on rrd persistence doesn't seem right.